boards/jiminy-mega256rfr2/doc.txt

```diff
@@ -133,4 +133,36 @@ This should take care of everything!
 
 RIOT's Makefile are configured to flash the jiminy using AVRDUDE. The bootloader automatically matches
 to the configured baud rate which is set for AVRDUDE. Rates of up to 500kBaud can be used.
- */
\ No newline at end of file
+
+# Fuse Settings
+Reading out the fuses can be done with
+`avrdude -c wiring -p m256rfr2 -P /dev/ttyACM0 -b 0010005 -U lfuse:r:-:i -v`
+
+The last line should read 
+`avrdude: safemode: Fuses OK (E:FE, H:D0, L:E2)`
+
+Which describes the fuses as follows:
+E: extended fuse = FE
+H: high fuse     = D0
+L: low fuse      = E2
+
+To program this fuse setting execute following line
+`avrdude -c wiring -p m256rfr2 -P /dev/ttyACM0 -b 0010005 -U lfuse:w:0xe2:m -U hfuse:w:0xd0:m -U efuse:w:0xfe:m`
+
+# Troubleshooting
+
+## Using 1.8V as board Voltage
+
+Setting the TPS6274x output voltage to 1.8V may lead to an undervoltage and thus triggers the brown-out reset.
+Disabling the brown-out detection by setting the extended fuse bit is recommended when a 1.8V supply is used.
+
+`avrdude -c wiring -p m256rfr2 -P /dev/ttyACM0 -b 0010005 -U efuse:w:0xff:m`
+
+## Using the external crystal oscillator (Transceiver Crystal Oscillator) and deep sleep
+
+When the external crystal oscillator is used as system clock and the device is put into deep sleep mode it seems that
+the clocks for all peripherals are enabled and set to the smallest devider (highest frequency). This leads to a higher
```

Comment:
devider-> divider

Reply:
Not fixed yet. Is it?

Reply:
Well, I addressed it and directly pushed to your branch. Guess I should've asked, but it happened more when testing this functionality – sorry for that.